app/models/conversion/project.rb

```diff
@@ -19,4 +19,8 @@ def fetch_provisional_conversion_date
   def all_conditions_met?
     tasks_data.conditions_met_confirm_all_conditions_met?
   end
+
+  def opened?
+    !conversion_date_provisional? && conversion_date.past?
```

Comment:
I know this is super pedantic! 

Can we change this to `conversion_date_confirmed_and_passed?` or something - I am nervous about this whole 'has the school opened' thing and I would rather it was not baked into the code until we get a straight answer.